firefox-ios/Client/Frontend/Settings/AppearanceSettings/Generic views/GenericSectionView.swift

```diff
@@ -55,7 +55,7 @@ struct GenericSectionView<Content: View>: View {
             Divider().frame(height: UX.dividerHeight)
 
             content()
-                .padding([.top, .bottom], UX.contentPadding)
+                // .padding([.top, .bottom], UX.contentPadding)
```

Comment:
If it's commented it can be removed?

Reply:
No idea what happened. Lesson learned never use xcode thingy for commits again. Back to terminal lol